DashAI/back/dataloaders/classes/dashai_dataset.py

```diff
@@ -120,6 +121,49 @@ def change_columns_type(self, column_types: Dict[str, str]):
         dataset = self.cast(new_features)
         return dataset
 
+    def sample(
+        self,
+        n: int = 1,
+        method: Literal["head", "tail", "random"] = "head",
+        seed: int | None = None,
```

Comment:
This notation is not compatible with python 3.8 :((

You have to change it to:

```suggestion
        seed: Union[int, None] = None,
```

And import as `from typing import Union`